var/spack/repos/builtin/packages/octopus/package.py

```diff
@@ -205,6 +211,8 @@ def configure_args(self):
 
         # --with-etsf-io-prefix=
         # --with-sparskit=${prefix}/lib/libskit.a
```

Comment:
```suggestion
```
Suggest to delete the comment which presumably was meant to remind us to add the --sparskit variant.